src/drivers/windows/lv_windows_display.c

```diff
@@ -111,10 +111,74 @@ int32_t lv_windows_dpi_to_physical(int32_t logical, int32_t dpi)
     return MulDiv(logical, dpi, USER_DEFAULT_SCREEN_DPI);
 }
 
+void lv_windows_set_top_level(lv_display_t * display, bool top_level)
+{
+    HWND window_handle = lv_windows_get_display_window_handle(display);
+
+    LONG_PTR ex_style = GetWindowLongPtr(window_handle, GWL_EXSTYLE);
+    if(top_level) {
+        ex_style |= WS_EX_TOPMOST;
+        ex_style |= WS_EX_APPWINDOW;
+    }
+    else {
+        ex_style &= ~WS_EX_TOPMOST;
+    }
+
+    SetWindowLongPtr(window_handle, GWL_EXSTYLE, ex_style);
+    SetWindowPos(window_handle, top_level ? HWND_TOPMOST : HWND_NOTOPMOST, 0, 0, 0, 0,
+                 SWP_NOMOVE | SWP_NOSIZE | SWP_NOACTIVATE | SWP_FRAMECHANGED);
+}
+
+void lv_windows_set_frameless(lv_display_t * display, bool frameless)
+{
+    HWND window_handle = lv_windows_get_display_window_handle(display);
+
+    LONG_PTR style = GetWindowLongPtr(window_handle, GWL_STYLE);
+    if(frameless) {
+        style &= ~WS_OVERLAPPEDWINDOW;
+        style |= WS_POPUPWINDOW;
+    }
+    else {
+        style &= ~WS_POPUP;
+        style |= WS_OVERLAPPEDWINDOW;
+    }
+
+    SetWindowLongPtr(window_handle, GWL_STYLE, style);
+    SetWindowPos(window_handle, NULL, 0, 0, 0, 0, SWP_NOMOVE | SWP_NOSIZE | SWP_NOZORDER | SWP_FRAMECHANGED);
+}
+
 /**********************
  *   STATIC FUNCTIONS
  **********************/
 
+static int lv_windows_center_window(HWND hwnd)
```

Comment:
Maybe we should use window_handle instead of hwnd to improve readability and keep the coding style.

Also, some variables like windowWidth should use snake case like window_width to keep the coding style.